### DISCUSSION TITLE
Make ordering reading lists easier

### DISCUSSION THREAD

**Discussion by djeebus** [gotson/komga#1584] [Ideas]
### Is your feature request related to a problem? Please describe.
No

### Describe the solution you'd like
Sorting large (100+ books) reading lists is currently a chore. There are a few things that could make this easier:
1. ~~Support scrolling + dragging at the same time, or allow draggin off the page to scroll up or down~~
2. ~~Add the series name, in addition to the issue number and title. Some issues don't have names, and when using reading lists to merge meld series together, it currently requires zooming in and reading the cover image, which is sometimes tough.~~
3. Support editing as a table, or list, rather than the cards that flow left to right in addition to up and down. This is the largest change, and would allow for displaying series name, title, issue number, publication date, etc.

### Describe alternatives you've considered or other apps that can do what you want
I tried to use the current UI. It's doable for smaller reading lists, but it's pretty rough for larger ones. I was also going to try and use the API directly, but that seems ... less useful for the community at large.

### Additional context
I was going to try and implement the second item on my own, but it turns out ... I don't know vue :) I'm still trying, but I suspect it'll be much faster for you all to do it.

Great project, thanks a ton!

**Comment by gotson** [comment]
I've added a text field when in edit mode, with the current index. Changing the index will move the item to the new position, a bit like how the Steam Wishlist works.

Would that address your point 3 ?

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.19.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.19.0))
